Content.Shared/Roles/JobRequirement/RoleTimeRequirement.cs

```diff
@@ -56,7 +59,7 @@ public override bool Check(IEntityManager entManager,
             reason = FormattedMessage.FromMarkupPermissive(Loc.GetString(
                 "role-timer-role-insufficient",
                 ("time", formattedRoleDiff),
-                ("job", Loc.GetString(proto)),
+                ("job", Loc.GetString(indexedJob.Name)),
```

Comment:
```suggestion
                ("job", indexedJob.LocalizedName),
```
Just use the property, makes it easier in case the component is changed later.